pkg/artifactory/resource/configuration/resource_artifactory_property_set.go

```diff
@@ -0,0 +1,290 @@
+package configuration
+
+import (
+	"context"
+	"github.com/jfrog/terraform-provider-shared/packer"
```

Comment:
Please group the import statements in the following order:

1. built-in
2. app internal packages
3. 3rd party packages

So in this case, all the 3rd party packages should be grouped together and sorted alphabetically. Separate from the Golang packages with single blank line.

Reply:
Fixed